### PR TITLE
Fix add jhipster-needle-add-admin-route

### PR DESCRIPTION
**Pull request by DanielFran**
Tested using current generator v6.10.4

### CODE REVIEW

**Comment by DanielFran** [comment]
@murdos Actually to have it working it is needed this fix and also PR #147 and #157

File: generators/app/index.js [hunk 1]
```diff
@@ -424,7 +424,7 @@ module.exports = class extends BaseGenerator {
             this.rewriteFile(
               adminModulePath,
               'jhipster-needle-add-admin-route',
-              ',\n      {\n        path: \'entity-audit\',\n        loadChildren: () => import(\'./entity-audit/entity-audit.module\').then(m => m.EntityAuditModule)\n      }'
+              '      {\n        path: \'entity-audit\',\n        loadChildren: () => import(\'./entity-audit/entity-audit.module\').then(m => m.EntityAuditModule)\n      }'
```
Comment (mshima) [inline generators/app/index.js:427]: Keep comma for next needle.
```suggestion
              '      {\n        path: \'entity-audit\',\n        loadChildren: () => import(\'./entity-audit/entity-audit.module\').then(m => m.EntityAuditModule)\n      },'
```

Comment (DanielFran) [reply generators/app/index.js:427]: @mshima It is already adding it, but I do not know how:

image

Comment (mshima) [reply generators/app/index.js:427]: Prettier is adding.
But it will fail if add a second needle is added.

Comment (DanielFran) [reply generators/app/index.js:427]: @mshima Will regenerate a new app and confirm

Comment (mshima) [reply generators/app/index.js:427]: It's a module, prettier probably will be executed before any other interaction.

**Review by mshima — APPROVED** [review]

**Pull request merged**